Changelog.md

```diff
@@ -2,7 +2,44 @@
 
 ### Ignition Gui 4.X.X
 
-### Ignition Gui 4.0.0 (20XX-XX-XX)
+### Ignition Gui 4.X.X (20XX-XX-XX)
```

Comment:
Perhaps this is a misunderstanding from my end, but why are we moving from `4.0.0` to `4.X.X`? `4.0.0` is still being used a few lines below.

Reply:
Oh, this is just a placeholder here for the next minor / patch release. People will often add entries below the `N.X.X` before the release